README.md

```diff
@@ -1,2 +1,21 @@
-# okta-jwt-elixir
-Okta JWT verification for elixir
+# OktaJwtElixir
+
+**TODO: Add description**
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `okta_jwt_elixir` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:okta_jwt_elixir, "~> 0.1.0"}
+  ]
+end
```

Comment:
I'll set up hex pm and publish this package when I've updated the http client and added tests.